stdlib/public/TensorFlow/NumpyConversion.swift

```diff
@@ -108,4 +110,21 @@ extension Tensor : ConvertibleFromNumpyArray
     self.init(shape: shape, scalars: buffPtr)
   }
 }
+
+extension Tensor where Scalar : NumpyScalarCompatible {
+  /// Constructs a matching numpy ndarray (with copy) from a Tensor.
+  public func makeNumpyArray() -> PythonObject {
+    let status = TF_NewStatus()
+    let cTensor = TFE_TensorHandleResolve(self.handle._cTensorHandle, status)
+    TF_DeleteStatus(status)
+
+    let data = ctypes.cast(Python.int(Int(bitPattern: TF_TensorData(cTensor))),
```

Comment:
Using `ctypes` and C pointer bridging is unnecessary, because the new array isn't going to share the underlying buffer anyway. All this functionality can be implemented with roughly a one-liner:

```swift
return np.array(scalars).reshape(Array(shape))
```